pkg/datapath/linux/config/config.go

```diff
@@ -278,6 +279,10 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 
 	if option.Config.EnableIPSec {
 		cDefinesMap["ENABLE_IPSEC"] = "1"
+
+		if option.Config.EnableIPSecEncryptedOverlay {
+			cDefinesMap["ENABLE_ENCRYPTED_OVERLAY"] = "1"
+		}
```

Comment:
We don't have any compile coverage for the affected paths atm, right? Could you please wire up `bpf/Makefile`?

Reply:
fixed in latest push.